docker/Dockerfile.deps

```diff
@@ -243,8 +241,5 @@ RUN LIBSND_VERSION=1.0.28 && cd /tmp
     cd / && \
     rm -rf /tmp/libsndfile-$LIBSND_VERSION
 
```

Comment:
I thought the idea was to separate manylinux+CUDA from the other third party libraries we need

Reply:
That is true. docker/Dockerfile.cuda.deps is for that.
You can either:
- run docker/Dockerfile.cuda.deps with manylinux as a base and have just manylinux+cuda (other users)
- run docker/Dockerfile.deps with manylinux as a base, create manylinux with the DALI deps, then use it as a base for docker/Dockerfile.cuda.deps and have just manylinux+DALI deps+cuda (DALI use case)